env.template

```diff
@@ -28,10 +28,12 @@ AIRFLOW_VAR_API_KEY_BROOKLYN_MUSEUM=not_set
 AIRFLOW_VAR_API_KEY_DATA_GOV=not_set
 AIRFLOW_VAR_API_KEY_EUROPEANA=not_set
 AIRFLOW_VAR_API_KEY_FLICKR=not_set
+AIRFLOW_VAR_API_KEY_FREESOUND=not_set
 AIRFLOW_VAR_API_KEY_JAMENDO=not_set
 AIRFLOW_VAR_API_KEY_NYPL=not_set
+AIRFLOW_VAR_API_KEY_RAWPIXEL=not_set
 AIRFLOW_VAR_API_KEY_THINGIVERSE=not_set
-AIRFLOW_VAR_API_KEY_FREESOUND=not_set
+AIRFLOW_VAR_API_KEY_WALTERS_ART_MUSEUM=not_set
```

Comment:
That's a great thought! I think it's worth having them here in the example `env.template` file just to show what they _would be_ (since Airflow Variables don't quite behave the same as normal env vars) but we can take the same strategy as API: define them but comment them out.

Reply:
Ah, okay I remember now why they're defined - they're [needed for tests 😅](https://github.com/WordPress/openverse-catalog/actions/runs/3301170179/jobs/5446361908) (via the [`just dotenv` recipe](https://github.com/WordPress/openverse-catalog/blob/516d7674be2a9e619b13a550ac611c82d413bccc/justfile#L41)). I'm going to revert this change for now, but create an issue for potentially using a plugin like [pytest-env](https://github.com/MobileDynasty/pytest-env) specifically for this in the future.